src/util.h

```diff
@@ -1299,34 +1299,6 @@ class MathUtils
 };
 
 
-// Precise time ----------------------------------------------------------------
-// required for ping measurement
-class CPreciseTime
```

Comment:
What about the precision of the QElapsedTimer? Have you tested the precision on all supported platforms (Windows, Linux and MacOS)? As far as I remember (it is a very long time ago when I implemented that), I think I experimented with the Qt internal classes but the precision was not good enough. That was the reason I have implemented my own class.

Reply:
I checked: src/corelib/kernel/qelapsedtimer_mac.cpp, src/corelib/kernel/qelapsedtimer_win.cpp

Mac implementation is the same like yours (nanoseconds) Windows is no worse either (milliseconds).

Reply:
Ok, so there was no issue with the functionality but you just want to improve the code. Maybe QElapsedTimer has improved since I tested it a long time ago. But I have to check the ping timer precision when I have access to my PC again (in about a week) to verify that the functionality does not changed by this replacement.

Reply:
Sure.